### PR TITLE
travis files, readme

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[![Build Status](https://travis-ci.org/OCA/OCB.png?branch=7.0)](https://travis-ci.org/OCA/OCB)
+[![Coverage Status](https://coveralls.io/repos/OCA/OCB/badge.png?branch=7.0)](https://coveralls.io/r/OCA/OCB?branch=master)
+
+[Original readme](README)
```
Comment (pedrobaeza) [inline README.md:4]: Can we include the content of the original README instead of making a link?

Comment (hbrunn) [inline README.md:4]: google says no. we can copy it verbatim though

**Comment by hbrunn** [comment]
that looks okay now

File: README.md (hunk 1 above)
Comment (pedrobaeza) [inline README.md:4]: Maybe include directive? Or is it blocked?

Comment (hbrunn) [inline README.md:4]: that's what I meant, sorry for being unclear. I tried that: https://github.com/hbrunn/OCB/commit/c5f0a5a15f5db9c6e0b0d1821a535510de8e5c79

Comment (pedrobaeza) [inline README.md:4]: OK, I understand. Let's keep this way and we see if someone proposes another solution.

:+1: (code review)

Regards.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+language: python
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="7.0" DATABASE="openerp_test"
```
Comment (bwrsandman) [inline .travis.yml]: Version is a bit redundant and it doesn't look used.

File: README.md (hunk 1 above)
Comment (bwrsandman) [inline README.md:4]: Why not rename README to README.md?

Comment (hbrunn) [inline README.md:4]: because we want to change as little as possible from upstream

File: .coveragerc [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Config file .coveragerc
+# adapt the include for your project
+
+[report]
+include =
+    */OCA/OCB/addons/*
+    */OCA/OCB/openerp/addons/*
```
Comment (bwrsandman) [inline .coveragerc]: Is there a reason why we test only the addons and not the server?

Comment (bwrsandman) [inline .coveragerc]: Reporting on the contents of `*/OCB/openerp/*` could be helpful and should encourage devs to include tests when they fix certain functions.

**Comment by bwrsandman** [comment]
How about running flake8 on the changed lines using `flake8 --diff` and `git diff $TRAVIS_BRANCH` like it was proposed here:
https://github.com/OCA/OCB/pull/6/files#diff-354f30a63fb0907d4ad57269548329e3R29

We can't fix the entirety of the coding style of odoo, but we can at least make sure our changes are of superior quality.

**Comment by dreispt** [comment]
:+1: That's an interesting feature for tests on any pre-v8 branch.

File: .coveragerc (hunk 3 above)
Comment (hbrunn) [inline .coveragerc]: I think you're absolutely right

**Comment by hbrunn** [comment]
I'm not so sure about this one: I'd rather have consistently looking non-pep8-compliant code than compliant fixes that look weird when looking at the whole code

**Comment by yvaucher** [comment]
Agreed with @hbrunn as long as Odoo doesn't follow PEP8 and might give us none PEP8 patches, it is nonsense to check PEP8

:+1: for this PR as it is

**Comment by hbrunn** [comment]
It's been 3 weeks and 3 approvals, so I'll merge this in order to have proper tests for the PRs to come

**Pull request merged**